### ISSUE TITLE
Upgrade babel-plugin-relay to v15

### ISSUE THREAD

**Issue by michaelmcneilnet** [oscartbeaumont/vite-plugin-relay#477]
`babel-plugin-relay` v15 is available, but this library depends on v14.

```
ABC@workspace:. provides babel-plugin-relay@npm:15.0.0 with version 15.0.0, which doesn't satisfy the following requirements:
➤ YN0000: vite-plugin-relay@npm:2.0.0 [f5404] → ^14.1.0 ✘
```

Is it possible to upgrade this plugin to use v15? Is there another way to remove this warning from `yarn`?

**Comment by artola** [comment]
and 16 ?

would not be better in `dependencies` to define `^14.1 || ^15 || ^16`?

**Comment by artola** [comment]
@oscartbeaumont do you accept a PR for such change?

**Comment by oscartbeaumont** [comment]
Go for it. I have been meaning to do another Vite Plugin Relay release. Will hopefully do it in the next couple of weeks.

**Comment by tobias-tengler** [comment]
Sorry, I haven't seen this - I'm also a maintainer. If you make a PR, I can definitely get this in :)

**Comment by artola** [comment]
> Go for it. I have been meaning to do another Vite Plugin Relay release. Will hopefully do it in the next couple of weeks.

The change was already done (6-months ago by @koistya) but he package never released since then:

https://github.com/artola/vite-plugin-relay/commit/bcdac052686d3a30eda07be6a55bf16aaaefe3a9

As soon as you release a package, the problem is gone. Looking forward the NPM release.

**Comment by zeroliu** [comment]
Hello! I'm glad to see this problem will be solved soon. Is there a timeline for the next release? Thanks for the support!

**Comment by tobias-tengler** [comment]
I forgot that I'm also a maintainer on this repository 🙈 I just triggered [v2.1.0](https://github.com/oscartbeaumont/vite-plugin-relay/releases/tag/2.1.0) which includes the requested change. 

FYI @oscartbeaumont: I had to merge #496 & #497 in order to be able to trigger a release.
